src/counter/subaccounts.md

```diff
@@ -0,0 +1,101 @@
+# Deploying to a subaccount
+
+You might be wondering how to make a _named_ contract, like `counter.raendev.testnet` instead of `dev-1234-1234`. Some news for you:
+
+## They're all named contracts!
+
+If you're used to other blockchains, then you're familiar with the idea of a public key being synonymous with an account. Accounts and public/private keypairs are equivalent in blockchains like Ethereum and Bitcoin, but not NEAR!
+
+NEAR accounts _can_ copy this "public key as account name" setup. These are called [implicit accounts](https://docs.near.org/integrator/implicit-accounts). A neat thing about them is that you can create a unique, valid account address _before making any on-chain transactions_, entirely locally, using the magic of cryptography.
+
+But most NEAR accounts you'll interact with are named. Like your account, or `raendev.testnet`. Furthermore:
+
+- Any account can have subaccounts, like `counter.raendev.testnet`
+- Any account can have a contract deployed to it
+
+Your `dev-1234-1234` contract isn't an implicit account. It's a named account, with a contract deployed to it. It just has a bad name! How can you give it a good name?
+
+## CLI Login
+
+In your command line, run the command:
+
+```bash
+near login
+```
+
+This will open up testnet NEAR Wallet in your browser, and request a _Full Access_ key for your account. Go ahead and say Yes.
+
+Now verify that you have a file in `~/.near-credentials/testnet`:
+
+```bash
+cat ~/.near-credentials/testnet/*.testnet.json
+```
+
+Great! Now you can use your named account from the command line.
+
+## Create a subaccount
+
+You could deploy the counter contract right to the account you just used with `near login`, but you may want to leave it free for a different contract later. Or you might just want to leave it contract-free forever!
+
+Let's create `counter.YOUR-ACCOUNT.testnet`, but with your actual account name instead of `YOUR-ACCOUNT`. To make the following commands copy-pastable, let's set up a bash variable. Note that this is using bash syntax, which only works with bash, zsh, and a couple other unix-type shells. If you're using Windows, you may just want to modify all the commands that follow.
+
+Otherwise, this is the only command that you need to modify:
+
+```bash
+export ACCOUNT=your-account.testnet
+```
+
+Ok, now you can create a subaccount:
+
+```bash
+near create-account counter.$ACCOUNT --masterAccount $ACCOUNT --initialBalance 2
+```
+
+This makes a call to the blockchain as `your-account.testnet` (that's the `masterAccount` part), creating a subaccount, `counter.your-account.testnet`. It sends 2 NEAR tokens from `your-account.testnet` to the new account.
```

Comment:
I think we should increase this to at least 6 N. It's the minimum an account needs in order to deploy, according to astro.  Though we should test this!

Reply:
I've tested it with Counter an 2 was enough. The default for new accounts on testnet these days is only 10N! Blowing 6 of it on a `counter.` subaccount seems unfrugal.